tox.ini

```diff
@@ -45,7 +45,7 @@ commands =
 install_command = pip install --pre {opts} {packages}
 deps = black
 commands =
-    python -m black --check {posargs:.}
+    python -m black --check --diff {posargs:.}
```

Comment:
I don't think this is a good idea. It is better to knew that format is failing somewhere else.

If there is something failing it should be fixed in another PR.